cli/sawtooth_cli/main.py

```diff
@@ -133,6 +137,9 @@ def main_wrapper():
     # pylint: disable=bare-except
     try:
         main()
+    except AssertionError as e:
```

Comment:
AssertionError is used to indicate a code bug, and thus it should fall through to the bare except which prints out the stack trace.  It should be impossible for users to cause this.